### PR TITLE
Update overview of installing on linux Introduction section to make i…

### PR DESCRIPTION
**Pull request by rayzhou2017**
…t more clear logically.

Signed-off-by: Ray Zhou <ray@yunify.com>

### CODE REVIEW

File: content/en/docs/installing-on-linux/introduction/intro.md [hunk 1]
```diff
@@ -81,13 +60,9 @@ KubeSphere has decoupled some core feature components since v2.1.0. These compon
 
 ## Storage Configuration Instruction
 
-The following links explain how to configure different types of persistent storage services. Please refer to [Storage Configuration Instruction](../storage-configuration) for detailed instructions regarding how to configure the storage class in KubeSphere.
+KubeSphere allows you to configure persistent storage services before installation and after installation. Meanwhile KubeSphere supports a varity of storage services including open source solutions such as Ceph, GlusterFS, etc. and commercial storage products as well. Please refer to [Persistent Storage Configurations](../storage-configuration) for detailed instructions regarding how to configure the storage class before installing KubeSphere.
```
Comment (Sherlock113) [inline content/en/docs/installing-on-linux/introduction/intro.md:63]: Note that "etc." should never be used after "include", "such as" or "for example". These terms make etc. redundant as they already imply that the writer could offer other examples. Please change this paragraph to:

KubeSphere allows you to configure persistent storage services both before installation and after installation. Meanwhile, KubeSphere supports a variety of open-source storage solutions (for example, Ceph and GlusterFS) as well as commercial storage products. Please refer to [Persistent Storage Configurations](../storage-configuration) for detailed instructions regarding how to configure the storage class before you install KubeSphere.

File: content/en/docs/installing-on-linux/introduction/intro.md [hunk 2]
```diff
@@ -81,13 +60,9 @@ KubeSphere has decoupled some core feature components since v2.1.0. These compon
...
 
-- [NFS](https://kubernetes.io/docs/concepts/storage/volumes/#nfs)
-- [GlusterFS](https://www.gluster.org/)
-- [Ceph RBD](https://ceph.com/)
-- [QingCloud Block Storage](https://docs.qingcloud.com/product/storage/volume/)
-- [QingStor NeonSAN](https://docs.qingcloud.com/product/storage/volume/super_high_performance_shared_volume/)
+As said, You can set different storage classes after installing KubeSphere which can be used for your workloads. For more information, see [Add New Storage Classes](../../../cluster-administration/persistent-volume-and-storage-class/).
```
Comment (Sherlock113) [inline content/en/docs/installing-on-linux/introduction/intro.md:65]: ->
For more information about how to set different storage classes for your workloads after you install KubeSphere, see [Add New Storage Classes](../../../cluster-administration/persistent-volume-and-storage-class/).

File: content/en/docs/installing-on-linux/introduction/intro.md [hunk 3]
```diff
@@ -97,13 +72,7 @@ With KubeKey, you can scale the number of nodes to meet higher resource needs af
...
 ### Remove Nodes
 
-You need to drain a node before you remove. For more information, see Remove Nodes.
-
-### Add New Storage Classes
-
-KubeKey allows you to set a new storage class after the installation. You can set different storage classes for KubeSphere itself and your workloads.
-
-For more information, see Add New Storage Classes.
+You need to drain a node before you remove. For more information, see [Remove Nodes](../../../installing-on-linux/cluster-operation/remove-nodes/).
```
Comment (Sherlock113) [inline content/en/docs/installing-on-linux/introduction/intro.md:75]: You need to drain a node before you remove.
->
You need to drain a node before you remove it.

File: content/en/docs/installing-on-linux/introduction/intro.md (hunk 1 above)
Comment (rayzhou2017) [reply content/en/docs/installing-on-linux/introduction/intro.md]: done

File: content/en/docs/installing-on-linux/introduction/intro.md (hunk 3 above)
Comment (rayzhou2017) [reply content/en/docs/installing-on-linux/introduction/intro.md]: done

File: content/en/docs/installing-on-linux/introduction/intro.md (hunk 2 above)
Comment (rayzhou2017) [reply content/en/docs/installing-on-linux/introduction/intro.md]: done

**Pull request merged**